### DISCUSSION TITLE
Cadastro SICAF - Solicitação para Nota de Empenho da Python Cerrado

### DISCUSSION THREAD

**Discussion by anadulce** [apyb/comunidade#168] [Pergunta]
Uma instituição em trâmite para inscrição na Python Cerrrado via nota de empenho solicitou o cadastro no SICAF da APyB. Já temos este cadatro realizado? Poderiam me fornecer o número? Se não, poderiam criá-lo, por favor?

O cadastro no SICAF é obrigatório para qualquer organização, incluindo associações, que deseja fornecer produtos ou serviços ao governo federal. O cadastro é essencial para habilitar a APyB, realizar empenhos, pagamentos, etc. 

O acesso ao cadastro e ao sistema SICAF pode ser feito através do Portal Compras Governamentais, no endereço: [comprasgovernamentais.gov.br](https://comprasgovernamentais.gov.br/). O link direto para o SICAF é encontrado na seção "Acesso ao Sistema".

Caso haja alguma dúvida, ficamos à disposição para buscar mais informações perante à instituição solicitante.

@apyb/diretoria

**Comment by anicelysantos** [comment]
@anadulce vou vê com a contadora e te retorno.

**Reply by anadulce** [reply]
Muito obrigada, Any!! <3

**Comment by anicelysantos** [comment]
@anadulce não gerou nenhum número, mas peguei um certificado que evidencia nossa inscrição, deixo aqui em anexo.
[CRC - SICAF - APYB.pdf](https://github.com/user-attachments/files/17563320/CRC.-.SICAF.-.APYB.pdf)
